src/components/Map/index.js

```diff
@@ -9,6 +9,14 @@ const StyledViewerContainer = styled(ViewerContainer)`
   z-index: 400; // this elevation ensures that this container comes on top of the internal leaflet components
 `;
 
+const StyledMap = styled(MapComponent)`
+  cursor: default;
+
+  &.leaflet-drag-target {
+    cursor: all-scroll;
+  }
```

Comment:
Wouldn't `grab` and `grabbing` make more sense in this case?

Reply:
These are the requests of the designer.